dev-requirements.txt

```diff
@@ -1,4 +1,3 @@
--e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@master#egg=cloudify-dsl-parser==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-rest-client@master#egg=cloudify-rest-client==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-plugins-common@master#egg=cloudify-plugins-common==3.2a1
-nose
```

Comment:
This is what I get with nose included:

Double requirement given: nose (from -r dev-requirements.txt (line 5)) (already in nose, name='nose')
ERROR: could not install deps [nose, nose-cov, testfixtures, -rdev-requirements.txt, -rtest-requirements.txt]; v = InvocationError('/home/travis/build/cloudify-cosmo/cloudify-aws-plugin/.tox/py27/bin/pip install --pre nose nose-cov testfixtures -rdev-requirements.txt -rtest-requirements.txt (see /home/travis/build/cloudify-cosmo/cloudify-aws-plugin/.tox/py27/log/py27-1.log)', 1)
___________________________________ summary ____________________________________
ERROR:   py27: could not install deps [nose, nose-cov, testfixtures, -rdev-requirements.txt, -rtest-requirements.txt]; v = InvocationError('/home/travis/build/cloudify-cosmo/cloudify-aws-plugin/.tox/py27/bin/pip install --pre nose nose-cov testfixtures -rdev-requirements.txt -rtest-requirements.txt (see /home/travis/build/cloudify-cosmo/cloudify-aws-plugin/.tox/py27/log/py27-1.log)', 1)

I assume this is because this is in toxin:
[testenv:py27]
deps =
    nose
    nose-cov
    testfixtures
    -rdev-requirements.txt
    -rtest-requirements.txt
commands=nosetests --with-cov --cov-report term-missing --cov ec2 ec2/tests